spec/cypress/integration/accessibility.js

```diff
@@ -10,7 +10,7 @@ describe("Accessibility", () => {
 
     // School not registered page
     cy.get('[name="user[email]"]').type("doesntexist@example.com{enter}");
-    cy.get("h1").should("contain", "You do not have an account");
+    cy.title().should("equal", "Check your email - Early career framework");
```

Comment:
I love this

Reply:
I don't think we should be including ` - Early career framework` in every test

Reply:
Agreed, we'll need a helper